requirements_base.txt

```diff
@@ -4,7 +4,6 @@ h5py==2.6.0
 jupyter==1.0.0
 numpy==1.11.0
 Pillow==3.2.0
-pyosmium==2.6.0
```

Comment:
It's cool if we don't include it in the file, but we should probably add a comment explaining the dependencies that aren't included in requirements, so that it's documented at least (since people will expect to look in requirements for the python dependencies). 

Btw, I managed to install pyosmium with pip. See the last line of the first code block in my comment on issue 8 [here](https://github.com/trailbehind/DeepOSM/issues/8#issuecomment-219043280) for the magic incantation.